### PR TITLE
Add Download Option To All Datasets CU-cuj2gt

### PR DESCRIPTION
**Pull request by mahamzayed**
# Description

The purpose of this PR is to allow the download option in the `Get Dataset` dialog for all datasets. 

[Clickup Ticket](https://app.clickup.com/t/cuj2gt)
[Wrike Ticket](https://www.wrike.com/open.htm?id=529262647)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

1. Navigate to a dataset that is less than 5GB in size.
2. Click on the `Get Dataset` button. A dialog should open with an option to download the dataset aside from the AWS option. Clicking on the download button should download the dataset.
3. Navigate to a dataset that is more than 5GB in size.
4. Click on the `Get Dataset` button. A dialog should open with messaging on how the dataset is too large and no download button. You should still see the AWS instructions.
5. Viewing the dialog in multiple viewports should maintain the size similar to how it appears on Discover.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: components/DownloadDataset/DownloadDataset.vue [hunk 1]
```diff
@@ -53,22 +56,16 @@
...
-            </bf-button>
-          </nuxt-link>
-          <bf-button class="secondary button-spacing" @click="closeDialog">
-            Okay
+        <div class="disclosure-text-block">
+          <p>*Requester pays means that any costs associated with downloading the data will be charged to your AWS account.
+            For transfer pricing information, visit the <a href="https://aws.amazon.com/s3/pricing/" target="blank">AWS Pricing documentation.</a>
+          </p>
+          <div>
+            <bf-button class="secondary button-spacing" @click="closeDialog">
```
Comment (cameronbaney) [inline components/DownloadDataset/DownloadDataset.vue:64]: `bf-button` has been deprecated. Can this be replaced with `el-button`? I know there was a ticket to make all of the buttons consistent, and the solve was to use `el-button`, but I know that ticket hasn't been done yet (in favor of just using the design system.)
```suggestion
            <el-button class="secondary" @click="closeDialog">
```

**Review by cameronbaney — COMMENTED** [review]
The test script passes and the functionality looks good.

The "Copy functionality allows users to copy the AWS information to the clipboard" feature is missing.

The help link is incorrect (looks like it needs to be updated from before.)

To confirm, does "maintain the size similar to how it appears on Discover" mean it will not be resized for mobile, and the below screenshot is correct? I see that mobile designs for this feature were not supplied. I also see that the responsive code for this dialog was removed in this PR.

localhost_3000_datasets_32_type=dataset(Pixel 2)

**Comment by mahamzayed** [comment]
> The test script passes and the functionality looks good.
> 
> The "Copy functionality allows users to copy the AWS information to the clipboard" feature is missing.
> 
> The help link is incorrect (looks like it needs to be updated from before.)
> 
> To confirm, does "maintain the size similar to how it appears on Discover" mean it will not be resized for mobile, and the below screenshot is correct? I see that mobile designs for this feature were not supplied. I also see that the responsive code for this dialog was removed in this PR.
> 
> localhost_3000_datasets_32_type=dataset(Pixel 2)

Designs were not supplied regarding mobile functionality, and I was going to reach out to Jeremy/Leo to see if they could reach out to the team to provide mobile designs if they are applicable. I left it similar to what it looks like in Discover in the meantime, as the previous responsive css code was messing things up.

Thanks for the other feedback, will address!

**Comment by mahamzayed** [comment]
@cameronbaney I've added the logic for copy functionality, but this PR will be blocked until I can get the copy icon from the SPARC team. Furthermore, speaking with Jeremy it seems that the team may not want downloading entirely on mobile, but he will get a confirmation on Wednesday. Blocking this ticket in Clickup for now until those two conditions are met.

**Comment by mahamzayed** [comment]
@cameronbaney Leo and Jeremy confirmed with the SPARC team that the mobile designs will remain the same as the desktop designs so no changes need to be done there. I've also added the copy functionality that was missing. Will be looping in Joost or Leo to approve since these changes need to be out before the weekend.

**Review by cameronbaney — APPROVED** [review]
A PNG for the copy icon is certainly not ideal but I'm sure it's all you were supplied with. We can address this later on when we get the svg.

**Pull request merged**